docs/setup.md

```diff
@@ -3,44 +3,21 @@
 
 # Installation
 
-1. To install a widget, click **Content Hub** > **Discover**.
+> [!NOTE]  
+> The **FortiAI Configuration** widget installs with the <em>FortiAI</em> solution pack and hence may not require separate installation.
 
-2. From the list of solution pack that appears, search for and select **FortiAI**. 
+1. Click **Content Hub** > **Discover**.
 
-3. Click the card of the **FortiAI** solution pack card.
+2. Search **FortiAI Configuration** in the list of widgets that appears. 
 
-4. Click **Configure** on the bottom to begin configuration.
+3. Click the **FortiAI Configuration** widget card.
 
 # Configuration
 
-## FortiAI Configuration Wizard Widget Settings
+> [!NOTE]  
+> The **FortiAI Configuration** launches when configuring <em>FortiAI</em> solution pack and does not require an independent configuration.
```

Comment:
The configuration is explained in detail in solutions pack guide. Pleae give link to it here.